src/Components/Card/types.ts

```diff
@@ -0,0 +1,6 @@
+import { ITodo } from "../../view/types";
+
+export interface CardProps {
+    todo: ITodo['todo']
```

Comment:
This means that will be just a string then? You can put just string

Reply:
Yeah, I'm kind of stuck when it comes to this. I have an interface called IInput, in which contains the "todo" as in "todo: string", and since I wanted to use the "todo" in my Card file, I figured I should create an interface in which the todo refers to the todo I created back in the IInput interface. I don't know if I'm making any sense, but I hope you managed to understand me. And btw, thanks Vic!